src/runtime.rs

```diff
@@ -584,6 +584,10 @@ pub fn noop_program(_: &Pubkey, _: &[AccountInfo], _: &[u8]) -> ProgramResult {
 
 #[async_trait]
 impl SolanaRpcClient for TestRuntime {
+    fn as_any(&self) -> &dyn std::any::Any {
```

Comment:
Why do you need to use it as Any? And is a method required for that? Is it not possible to do this in client code?